code/script.js

```diff
@@ -1,53 +1,209 @@
 // DOM selectors (variables that point to selected DOM elements) goes here 👇
-const chat = document.getElementById('chat')
+
+const chat = document.getElementById("chat");
+const helpButton = document.getElementById("helpButton");
+const nameForm = document.getElementById("name-form");
 
 // Functions goes here 👇
 
 // A function that will add a chat bubble in the correct place based on who the sender is
 const showMessage = (message, sender) => {
   // The if statement checks if the sender is the user and if that's the case it inserts
   // an HTML section inside the chat with the posted message from the user
-  if (sender === 'user') {
+  if (sender === "user") {
     chat.innerHTML += `
       <section class="user-msg">
         <div class="bubble user-bubble">
           <p>${message}</p>
         </div>
         <img src="assets/user.png" alt="User" />  
       </section>
-    `
+    `;
     // The else if statement checks if the sender is the bot and if that's the case it inserts
     // an HTML section inside the chat with the posted message from the bot
-  } else if (sender === 'bot') {
+  } else if (sender === "bot") {
     chat.innerHTML += `
       <section class="bot-msg">
         <img src="assets/bot.png" alt="Bot" />
         <div class="bubble bot-bubble">
           <p>${message}</p>
         </div>
       </section>
-    `
+    `;
   }
 
   // This little thing makes the chat scroll to the last message when there are too many to
   // be shown in the chat box
-  chat.scrollTop = chat.scrollHeight
-}
+  chat.scrollTop = chat.scrollHeight;
+};
+
+const userMessage = (message) => {
+  showMessage(message, "user");
+};
+
+const botMessage = (message) => {
+  showMessage(message, "bot");
+};
 
 // A function to start the conversation
+
 const greetUser = () => {
   // Here we call the function showMessage, that we declared earlier with the argument:
   // "Hello there, what's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, what's your name?", 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆 and see what happens
-}
+  botMessage("Welcome to PlantBot. My name is Palm! What's your name?");
+  helpButton.remove();
+  getUserName();
+};
 
-// Eventlisteners goes here 👇
+const getUserName = () => {
+  nameForm.innerHTML = `<input id="name-input" type="text"/>
+  <button class="send-btn" id="sendButton" type="button">Send
+  </button>`;
+  const sendButton = document.getElementById("sendButton");
+  const nameInput = document.getElementById("name-input");
+  sendButton.addEventListener("click", () => returnUserName(nameInput));
+};
```

Comment:
The button works perfectly, but if you instead of clicking the button, press Enter, the form submits and the page reloads. I'd add an eventListener to the form as well, in order to prevent this default behaviour - and send the user to the next function.